src/main/java/edu/mbrooks/advancedjava/main/Collectable.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Program Name:    Collectable.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the collectable interface
+ */
+package edu.mbrooks.advancedjava.main;
+
+/**
+ *
+ * @author michaelbrooks
+ * This is a collection of something
+ */
+public class Collectable {
```

Comment:
Name should be specific

Reply:
Comment is bad...  This is a single collectable item - the base class for a collection of things, or a collectables - the derived class will be a collection of stamps, coins, porcelain figures, fishing poles, records, etc. 

I think it meets the "is a" rule:
          A Stamp "is a" Collectable.
          A Coin "is a" Collectable.
          A Baseball Card "is a" Collectable.

Do I have that wrong?